beamer/tests/agent/test_challenge.py

```diff
@@ -491,9 +571,99 @@ def test_invalidation(
         while agent.get_context(direction).l1_resolutions.get(claim.invalidation_tx) is None:
             sleeper.sleep(0.1)
 
+    ape.chain.provider._make_request(
+        "evm_setAccountBalance",
+        [l1_messenger.address, encode_hex(str(ape.convert("1 ether", int)))],
+    )
+
```

Comment:
Can we avoid using the private `_make_request` function here?

Reply:
oo sorry, actually l1_messenger is contract that has no payable so i cant randomly send funds to it with test account